python/cudf/cudf/tests/test_string.py

```diff
@@ -1,13 +1,14 @@
 # Copyright (c) 2018, NVIDIA CORPORATION.
```

Comment:
```suggestion
# Copyright (c) 2018-2020, NVIDIA CORPORATION.
```
Please also check copyright year of other files in the PR.

Reply:
Perhaps we should do this for all files in a separate PR? This seems completely orthogonal unless I'm missing something.

Reply:
The copyright year is changing because the file is being modified. If you are updating the file in 2020, it makes sense to update the copyright year to include the year in which you made those updates. If not this PR, then what PR?

Reply:
OK I've modified the copyright year for all files in this PR that had a header. What about those that don't? Do I just add `# Copyright (c) 2020, NVIDIA CORPORATION.` to those?

Reply:
I guess my concern is: let's say some time in the future we want to undo the changes introduced in this PR. This will also undo the copyright change which is a completely orthogonal side effect.